CHANGELOG-0.3.0.md

```diff
@@ -0,0 +1,18 @@
+# v0.3.0
```

Comment:
Can you actually make the filename based on minor versions, not patch verisons? So CHANGELOG-0.3.md

Reply:
Do we want rename existing changelog files too?

Reply:
yeah that sounds good